pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -30842,7 +30842,6 @@ self: {
        description = "Run docker-compose with help from Nix/NixOS";
        license = stdenv.lib.licenses.asl20;
        hydraPlatforms = stdenv.lib.platforms.none;
-       broken = true;
```

Comment:
This file is generated automatically, so if you edit it, there is a chance it will cause a build conflict.

Can you remove this change?  It will be marked as unbroken next time the file is automatically generated (which happens quite frequently).

You might be interested in this video which explains the process:

https://discourse.nixos.org/t/video-tutorial-how-to-fix-broken-haskell-packages-in-nix/3968

Reply:
How can you tell I didn't run it?

Tbh I got a linker error in cabal2nix. I wrote an issue here to make that process reproducible if you're interested: https://github.com/NixOS/cabal2nix/issues/431

Reply:
> How can you tell I didn't run it?

As far as I know, in general `hackage2nix` is run automatically by some sort of CI setup by peti.  So when people send PRs to nixpkgs, they are often asked to not change `pkgs/development/haskell-modules/hackage-packages.nix` by hand, because it will sometimes cause merge conflicts.

Also, `hackage2nix` will always(?) remove both the `broken = true;` _and_ `hydraPlatforms = stdenv.lib.platforms.none;` lines.